accounting/accounting_cli_functions.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python3
+
+###############################################################
+# Copyright 2020 Lawrence Livermore National Security, LLC
+# (c.f. AUTHORS, NOTICE.LLNS, COPYING)
+#
+# This file is part of the Flux resource manager framework.
+# For details, see https://github.com/flux-framework.
+#
+# SPDX-License-Identifier: LGPL-3.0
+###############################################################
+import sqlite3
+import argparse
+import time
+import sys
+
+import pandas as pd
+
+def view_user(user):
+    try:
+        # open connection to database
+        conn = sqlite3.connect("FluxAccounting.db")
+        # get the information pertaining to a user in the Accounting DB
+        select_stmt = (
+            "SELECT * FROM association_table where user_name='" + user + "'"
+        )
+        dataframe = pd.read_sql_query(select_stmt, conn)
+        # if the length of dataframe is 0, that means
+        # the user specified was not found in the table
+        if len(dataframe.index) == 0:
+            print("User not found in association_table")
+        else:
+            print(dataframe)
+    except pd.io.sql.DatabaseError as e_database_error:
+        print(e_database_error)
+
+def add_user(user_acct_info):
+    # open connection to database
+    conn = sqlite3.connect("FluxAccounting.db")
+
+    # insert the user values into the database
+    try:
+        conn.execute(
+            """
+            INSERT INTO association_table (
+                creation_time,
+                mod_time,
+                deleted,
+                user_name,
+                admin_level,
+                account,
+                parent_acct,
+                shares,
+                max_jobs,
+                max_wall_pj
+            )
+            VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?)
+            """,
+            (
+                int(time.time()),
+                int(time.time()),
+                0,
+                user_acct_info[0],
+                user_acct_info[1],
+                user_acct_info[2],
+                user_acct_info[3],
+                user_acct_info[4],
+                user_acct_info[5],
+                user_acct_info[6],
+            ),
+        )
+        # commit changes
+        conn.commit()
+    # make sure entry is unique
+    except sqlite3.IntegrityError as integrity_error:
+        print(integrity_error)
+    # make sure all fields are entered
+    except IndexError:
```

Comment:
perhaps this could be checked earlier on as an user input error, so that we don't have to make a connection to the sql database to produce the error message.